### DISCUSSION TITLE
如何把模型放在gpu上？

### DISCUSSION THREAD

**Discussion by ALR-alr** [RapidAI/RapidLatexOCR#25] [Q&A]
cpu推理太慢了，三分钟一张，有什么办法吗？

**Comment by SWHL** [comment]
这个后期我会出一版 基于torch 作为推理引擎的版本，可以直接用PyTorch推理
欢迎PR哈

**Reply by ALR-alr** [reply]
好的，谢谢

**Comment by sangeren** [comment]
> cpu推理太慢了，三分钟一张，有什么办法吗？

用C# 写的，400ms小的图片，大的1s左右。

**Reply by SWHL** [reply]
欢迎贡献C#版代码
